ci-operator/config/openshift/rosa/openshift-rosa-master__e2e.yaml

```diff
@@ -29,6 +29,28 @@ tests:
     test:
     - ref: rosa-test-e2e
     workflow: rosa-test-profile
+- as: rosa-advanced-f3
+  cron: 45 1 1,4,7,10,13,16,19,22,25,28 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: stable
+      TEST_LABEL_FILTERS: (day1-post,day2,day1)&&!NonClassicCluster&&!Exclude
```

Comment:
Please write the filter `(day1-post,day2,day1)` to `(day1, day1-post,day2)`. What does `day1` do? Is not `day1` for the cluster creation?

Reply:
@yasun1 In current testing, there are some tests using Day- 1, like account-roles, token command. The cluster creation is using day1-prepare

Reply:
day1 cases means they are running not based on an existing cluster if I am not making a mistake. Since we are not having so many cases till now it's OK to include the cases in 1 job. But for future if the job is running too many cases we have to split the day1 cases into another job.